src/main/java/com/vinylteam/vinyl/service/impl/DefaultEmailConfirmationService.java

```diff
@@ -20,7 +21,7 @@
 public class DefaultEmailConfirmationService implements EmailConfirmationService {
```

Comment:
Boyscout rule:

    @Override
    public UUID generateConfirmationToken() {
        UUID uuid = UUID.randomUUID();
        return uuid;
    }

remove variable declaration and just return UUID.randomUUID() please